johnslist/dbtest/static/org_search.js

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
In develop, under static, I created a JS directory. This will need to be moved into that directory.